tools/scalar-schema/src/scalar_schema/core.clj

```diff
@@ -24,6 +24,7 @@
    [nil "--region REGION" "Region where the tool creates tables for DynamoDB"]
    [nil "--prefix NAMESPACE_PREFIX" "Namespace prefix. The prefix is added to all the namespaces."]
    [nil "--no-scaling" "Disable auto-scaling"]
+   [nil "--no-backup" "Disable continuous backup"]
```

Comment:
Just a question. This option can only be specified for DynamoDB at the moment and would be ignored if specified for other databases?

Reply:
@feeblefakie Thank you for the question. Right, it will be ignored for other databases.